documentation/tut/toy-example-from-scratch.rst

```diff
@@ -181,7 +181,7 @@ Yes, that is quite a large battery :)
 We can also look at the battery asset in the UI of FlexMeasures (in Docker, the FlexMeasures web server already runs, on your PC you can start it with ``flexmeasures run``).
 Visit `http://localhost:5000/assets <http://localhost:5000/assets>`_ (username is "toy-user@flexmeasures.io", password is "toy-password") and select "toy-battery":
 
-.. image:: https://github.com/FlexMeasures/screenshots/raw/main/tut/toy-schedule/asset-view.png
+.. image:: https://github.com/FlexMeasures/screenshots/raw/main/tut/toy-schedule/asset-view-without-data.png
```

Comment:
Note: I realize this one renaming I proposed in https://github.com/FlexMeasures/screenshots/pull/1 is not correct.  

`asset-view-with-data.png` should be renamed `asset-view.png`

Reply:
I have replaced `asset-view-without-data.png` by another screenshot 'asset overview`, though it's name will be `asset-view.png` and I think  `asset-view-with-data.png` should be there because we are showing it after adding the data.